spec/Overview.html

```diff
@@ -870,6 +881,85 @@ <h4>The getRandomValues method</h4>
+            <ol>
+              <li>
+                Let |bytes| be a [= byte sequence =] of length 16.
+              </li>
+              <li>
+                Fill |bytes| with cryptographically secure random bytes.
+              </li>
+              <li>
+                Set the 4 most significant bits of |bytes|[6], which represent the
+                UUID <a data-cite="RFC4122#section-4.1.3">version</a>, to `0b0100`.
+              </li>
+              <li>
+                Set the 2 most significant bits of |bytes|[8], which represent the
+                UUID <a data-cite="RFC4122#section-4.1.1">variant</a>, to `0b10`.
+              </li>
+              <li>
+                <p>
+                  Return the [= string/concatenate | string concatenation =] of «
+                </p>
+                <ol style="list-style-type: none">
+                  <li>
+                    [= hexadecimal representation =] of |bytes|[0], [= hexadecimal
+                    representation =] of |bytes|[1], [= hexadecimal representation =] of
+                    |bytes|[2], [= hexadecimal representation =] of |bytes|[3],
+                  </li>
+                  <li>
+                    `"-"`,
+                  </li>
+                  <li>
+                    [= hexadecimal representation =] of |bytes|[4], [= hexadecimal
+                    representation =] of |bytes|[5],
+                  </li>
+                  <li>
+                    `"-"`,
+                  </li>
+                  <li>
+                    [= hexadecimal representation =] of |bytes|[6], [= hexadecimal
+                    representation =] of |bytes|[7],
+                  </li>
+                  <li>
+                    `"-"`,
+                  </li>
+                  <li>
+                    [= hexadecimal representation =] of |bytes|[8], [= hexadecimal
+                    representation =] of |bytes|[9],
+                  </li>
+                  <li>
+                    `"-"`,
+                  </li>
+                  <li>
+                    [= hexadecimal representation =] of |bytes|[10], [= hexadecimal
+                    representation =] of |bytes|[11], [= hexadecimal representation =] of
+                    |bytes|[12], [= hexadecimal representation =] of |bytes|[13],
+                    [= hexadecimal representation =] of |bytes|[14], [= hexadecimal
+                    representation =] of |bytes|[15]
+                  </li>
+                </ol>
+                <p>
+                  ».
+                </p>
+              </li>
+            </ol>
+            <p>
+              For the steps described in the algorithm to [= generate a random UUID =],
+              the <dfn>hexadecimal representation</dfn> of a byte |value| is the
+              two-character string created by expressing |value| in hexadecimal using
+              [= ASCII lower hex digits =], left-padded with `"0"` to reach two
+              [= ASCII digits =].
```

Comment:
Also, while editing I noticed, this probably isn't quite accurate (since it's still two hex digits, not two 0-9 digits), so I think it should either say "ASCII lower hex digits" again, or just say "to reach two characters" or so?

Reply:
I think I like `ASCII lower hex digits`, just to be explicit (since two characters could imply a unicode or ascii character).